server/studentAuth.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const config = require("./utils/config");
+const pool = require('./db');
+const auth = require('./auth');
+
+const router = express.Router();
+
+router.post('/login', async (req, res) => {
+  const { username, password } = req.body;
+
+  try {
+    const result = await pool.query(
+      'SELECT id, password FROM users WHERE username = $1',
+      [username],
+    );
+    const user = result.rows[0];
+    if (!user) {
+      return res.status(401).json({ error: 'Invalid credentials' });
+    }
+    const passwordMatch = await bcrypt.compare(password, user.password);
+    if (!passwordMatch) {
+      return res.status(401).json({ error: 'Invalid credentials' });
+    }
```

Comment:
At the moment we are not using hashing out passwords, may be easier to just compare the passwords directly without using becrypt otherwise the rest of the logic looks good